README.rst

```diff
@@ -234,7 +234,7 @@ During RAKE initialization only language code should be used.
 - sv - Swedish
 - tr - Turkish
 - uk - Ukrainian
-
+- fa - Persian
```

Comment:
Could you, please, do it in alphabetic order? Between `es` and `fi`.

Reply:
sure